source/_posts/To-monitor-and-restart-services-using-Monit.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Nayan is a platform that offer :  this line is repeated in the bottom so please remove it from there.

Reply:
Sir it is not repeated. Can you please check it.

Reply:
Sir, I have resolved this issue